packages/sitecore-jss/src/i18n/mockDictionaryQueryResponse.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@anastasiya29 Can we create `sitecore-jss/testData` folder and put test related data there?